src/commands/cmd_timeseries.cc

```diff
@@ -51,6 +51,47 @@ std::string FormatAddResultAsRedisReply(TSChunk::AddResultWithTS res) {
   return "";
 }
 
+std::string_view FormatChunkTypeAsRedisReply(TimeSeriesMetadata::ChunkType chunk_type) {
+  using ChunkType = TimeSeriesMetadata::ChunkType;
+  static const std::unordered_map<ChunkType, std::string_view> map = {
+      {ChunkType::COMPRESSED, "compressed"},
+      {ChunkType::UNCOMPRESSED, "uncompressed"},
+  };
```

Comment:
Could you move them to global variables instead of local static variables?

Local static variables with non-trivial ctor will lead to a lock (i.e. `__cxa_guard_acquire`) to ensure thread-safety.